JustBrewIt/Pages/Search.cshtml

```diff
@@ -6,24 +6,18 @@
 
 <form action="" method="POST" style="margin-top: 50px;">
     <div class="form-group">
-        <label class="control-label" for="BreweryCity">Search Brewries by City</label>
+        <label class="control-label" for="BreweryCity">Enter City Name Here</label>
```

Comment:
Remove the word "here".  That's considered tacky, in web lingo.
Actually, the original prompt looks better to me, so I'd just leave it as it were.